src/widget/wsearchrelatedtracksmenu.cpp

```diff
@@ -95,7 +96,7 @@ QString WSearchRelatedTracksMenu::elideActionText(
         // This should never fail
         return actionTextPrefix;
     }
-    const auto actionTextPrefixWithSeparator =
+    const QString actionTextPrefixWithSeparator =
```

Comment:
This warning is a false positive if at least one argument is a QString as is here. But it doesn't hurt to be explicit.